### ISSUE TITLE
Multiple Subdomains

### ISSUE THREAD

**Issue by Meche697** [AlexHimself/duckdns-ps#1]
In the readme, it shows that you can run multiple updates to subdomains by adding a comma
Does that hold true for:
- token
- ip address

Will this task run as a service such that it does NOT matter which user is logged in?
I was thinking of creating another subdomains, on friends and the other family, but I'm having issues when I log in as a different user

**Comment by AlexHimself** [comment]
> Will this task run as a service such that it does NOT matter which user is logged in?

Yes, it should - [see this line](https://github.com/AlexHimself/duckdns-ps/blob/99417619df066d79c9c68891441d12f27bce6f65/DuckDNS-Updater.ps1#L89) and it runs as `NT AUTHORITY\SYSTEM`.

> In the readme, it shows that you can run multiple updates to subdomains by adding a comma
Does that hold true for:
> - token
> - ip address

I'm a little confused by this question, but it might be your implementation is something I hadn't considered. Are you planning, for example, to run the updater on _your_ machine on your network and have it send updates for other IPs that are not on your network?

If that's the case, it doesn't make sense to me to use this to continually update the IP's of external network machines with a static IP unless you have some way to detect their IP remotely? Otherwise, wouldn't you just type it in the web portal one time and leave it?
